tox.ini

```diff
@@ -172,8 +172,12 @@ envlist =
     pypy3-test-instrumentation-grpc
 
     ; opentelemetry-instrumentation-sqlalchemy
-    py3{8,9,10,11}-test-instrumentation-sqlalchemy-{14}
-    pypy3-test-instrumentation-sqlalchemy-{11,14}
+    ; The numbers at the end of the environment names
+    ; below mean these dependencies are being used:
+    ; 0: sqlalchemy>=1.1,<1.2 aiosqlite
```

Comment:
It looks like `aiosqlite` is in `requirements-1.txt`, not `0` right?

Reply:
right, fixed